homeworks/homework_2/src/components/Search.js

```diff
@@ -0,0 +1,91 @@
+import React, {Component} from 'react';
+import {Video} from "./Video";
+import {Results} from "./Results";
+import {Background} from "./Background";
+import YouTubeSearch from 'youtube-api-search';
+
+export class Search extends Component {
+    constructor (props) {
+        super(props)
+        this.state = {
+            videos: [],
+            api_key: 'AIzaSyAq8HPrbemKw4a23McQJD9ksl2w2lGAcII',
+            isMinimize: false,
```

Comment:
you don't need such state trigger the same as `resultShown`, `selectedThumb`, `selectedDesc`

Reply:
It could be done much easier. After all refactoring if you still have any questions how to drop such state properties feel free to DM me